resources/views/annotations/show.blade.php

```diff
@@ -64,11 +64,10 @@
             </div>
         </message-curtain>
         <annotation-canvas
-            v-show="!imageLoadingError"
-            :can-add="isEditor"
-            :can-modify="isEditor"
-            :can-delete="isEditor"
-            :image="image"
+            :can-add="canAdd"
+            :can-modify="canModify"
+            :can-delete="canDelete"
+            :image="!imageLoadingError ? image : null"
```

Comment:
With the proposed method above, `image` will be automatically `null` on error.

```suggestion
            :image="image"
```